### ISSUE TITLE
GridLayout

### ISSUE THREAD

**Issue by kidouchi** [codepath/android_guides#114]
Could we get an article regarding how/when to use these and best practices?

**Comment by nesquena** [comment]
Yes, great suggestion! GridLayout is one of missing areas of our guides.

**Comment by martincox** [comment]
Hey

I want to get familiar with contributing to OS projects on GitHub. I'm familiar with GridLayouts and would like to contribute a guide on the topic.

I've had a quick browse through the contribution guidelines. What's the next port of call? Crack on with writing the guide?

**Comment by nesquena** [comment]
Hi @martincox,

We'd love for you to contribute. In short, start by taking a look [at some of our existing guides](https://guides.codepath.com/android/Constructing-View-Layouts) and then [start a collaborative draft](https://hackmd.io/) so you can get feedback. Post the draft here when it's ready for review.

If you want to share an outline first, that can be helpful. I think it could either be a part of the guide linked above or it's own separate topic guide.

**Comment by ibtehaz-shawon** [comment]
Hi @nesquena 
I have written a draft over gridLayout. I am quite new in this and I think I have misused some English word. 
Here is the link : https://hackmd.io/CwDhAZgdgVhhaAnAQ3AZnsAxlYSQBmB8W4IyFMATAIwBsVBQA===?view
Also, I was unable to add the images so, I have provided the links to imgur. 
Please take a look and let me know. I will work on more upon review. 
Thank you :)

**Comment by nesquena** [comment]
@ibtehaz-shawon Thanks! Will help clean up the draft and move it over to the guides when it's ready. I fixed the images and a bit of formatting already but will need more cleanup. Great start though, thanks for putting this together

**Comment by ibtehaz-shawon** [comment]
@nesquena, should I need to do anything else? explain a bit more or more examples etc?
